webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala

```diff
@@ -69,10 +70,21 @@ import net.liftweb.http.js.JsCmds.*
 import scala.collection.MapView
 import scala.xml.*
 
-sealed trait ComplianceLevelPieChart {
-  def color: String
-  def label: String
-  def value: Int
+case class ScoreChart(scoreValue: ScoreValue, value: Int) {
+  def color: String = {
+
+    import com.normation.rudder.score.ScoreValue.*
+    scoreValue match {
+      case A       => "#13beb7"
+      case B       => "#68c96a"
+      case C       => "#b3d337"
+      case D       => "#fedc04"
+      case E       => "#f0940e"
+      case F       => "#da291c"
+      case NoScore => ""
```

Comment:
Why not grey ?

Reply:
In fact i think we want to filter Nodes without a score, but i don't know really. Yes in this case we should display them in grey